src/components/Calculator.js

```diff
@@ -1,35 +1,51 @@
-          <button type="button" id="x" value="x" className="special" onClick={null}>&times;</button>
-          <button type="button" id="4" value="4" onClick={null}>4</button>
-          <button type="button" id="5" value="5" onClick={null}>5</button>
-          <button type="button" id="6" value="6" onClick={null}>6</button>
-          <button type="button" id="-" value="-" className="special" onClick={null}>-</button>
-          <button type="button" id="1" value="1" onClick={null}>1</button>
-          <button type="button" id="2" value="2" onClick={null}>2</button>
-          <button type="button" id="3" value="3" onClick={null}>3</button>
-          <button type="button" id="+" value="+" className="special" onClick={null}>+</button>
-          <button type="button" id="3" value="3" className="zero" onClick={null}>0</button>
-          <button type="button" id="." value="." onClick={null}>.</button>
-          <button type="button" id="=" value="=" className="special" onClick={null}>=</button>
+          <button type="button" id="AC" value="AC" onClick={(e) => this.handleBtn(e.target.id)}>Ac</button>
+          <button type="button" id="+/-" value="+/-" onClick={(e) => this.handleBtn(e.target.id)}>+/-</button>
+          <button type="button" id="%" value="%" onClick={(e) => this.handleBtn(e.target.id)}>%</button>
+          <button type="button" id="&divide;" value="&divide;" className="special" onClick={(e) => this.handleBtn(e.target.id)}>&divide;</button>
+          <button type="button" id="7" value="7" onClick={(e) => this.handleBtn(e.target.id)}>7</button>
+          <button type="button" id="8" value="8" onClick={(e) => this.handleBtn(e.target.id)}>8</button>
+          <button type="button" id="9" value="9" onClick={(e) => this.handleBtn(e.target.id)}>9</button>
+          <button type="button" id="x" value="x" className="special" onClick={(e) => this.handleBtn(e.target.id)}>&times;</button>
+          <button type="button" id="4" value="4" onClick={(e) => this.handleBtn(e.target.id)}>4</button>
+          <button type="button" id="5" value="5" onClick={(e) => this.handleBtn(e.target.id)}>5</button>
+          <button type="button" id="6" value="6" onClick={(e) => this.handleBtn(e.target.id)}>6</button>
+          <button type="button" id="-" value="-" className="special" onClick={(e) => this.handleBtn(e.target.id)}>-</button>
+          <button type="button" id="1" value="1" onClick={(e) => this.handleBtn(e.target.id)}>1</button>
+          <button type="button" id="2" value="2" onClick={(e) => this.handleBtn(e.target.id)}>2</button>
+          <button type="button" id="3" value="3" onClick={(e) => this.handleBtn(e.target.id)}>3</button>
+          <button type="button" id="+" value="+" className="special" onClick={(e) => this.handleBtn(e.target.id)}>+</button>
+          <button type="button" id="3" value="3" className="zero" onClick={(e) => this.handleBtn(e.target.id)}>0</button>
```

Comment:
- [ ] Kindly make sure to fix the `0` button so it's value is right, for now the value is `3`

Reply:
Resolved